<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerPathExplorerModuleWidget</class>
 <widget class="qSlicerWidget" name="qSlicerPathExplorerModuleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>316</width>
    <height>211</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ctkCollapsibleButton" name="PlanningFrame">
     <property name="text">
      <string>Planning</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="qSlicerPathExplorerMarkupsTableWidget" name="EntryWidget" native="true"/>
      </item>
      <item>
       <widget class="qSlicerPathExplorerMarkupsTableWidget" name="TargetWidget" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="VisualizationFrame">
     <property name="text">
      <string>Visualization</string>
     </property>
     <property name="collapsed">
      <bool>true</bool>
     </property>
     <property name="contentsFrameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="qSlicerPathExplorerTrajectoryTableWidget" name="TrajectoryWidget" native="true"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="ReslicingLayout"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="AdvancedFrame">
     <property name="text">
      <string>Advanced</string>
     </property>
     <property name="collapsed">
      <bool>true</bool>
     </property>
     <property name="contentsFrameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="qMRMLNodeComboBox" name="EntryListSelector">
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLMarkupsFiducialNode</string>
         </stringlist>
        </property>
        <property name="baseName">
         <string>Entry</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="qMRMLNodeComboBox" name="TargetListSelector">
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLMarkupsFiducialNode</string>
         </stringlist>
        </property>
        <property name="baseName">
         <string>Target</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="qMRMLNodeComboBox" name="TrajectoryListSelector">
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLPathPlannerTrajectoryNode</string>
         </stringlist>
        </property>
        <property name="baseName">
         <string>Path</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qSlicerPathExplorerMarkupsTableWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerPathExplorerMarkupsTableWidget.h</header>
   <container>1</container>
   <slots>
    <slot>setMRMLScene(vtkMRMLScene*)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>qSlicerPathExplorerTrajectoryTableWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerPathExplorerTrajectoryTableWidget.h</header>
   <container>1</container>
   <slots>
    <slot>setMRMLScene(vtkMRMLScene*)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>qSlicerPathExplorerModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>EntryWidget</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>38</x>
     <y>7</y>
    </hint>
    <hint type="destinationlabel">
     <x>21</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerPathExplorerModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>TargetWidget</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>5</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>19</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerPathExplorerModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>TrajectoryWidget</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>4</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>21</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerPathExplorerModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>EntryListSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>309</y>
    </hint>
    <hint type="destinationlabel">
     <x>59</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerPathExplorerModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>TargetListSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>117</x>
     <y>310</y>
    </hint>
    <hint type="destinationlabel">
     <x>109</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerPathExplorerModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>TrajectoryListSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>195</x>
     <y>312</y>
    </hint>
    <hint type="destinationlabel">
     <x>170</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
